ml4gw/waveforms/cbc/coefficients.py

```diff
@@ -0,0 +1,32 @@
+import torch
+
+from ml4gw.constants import C, G
+from ml4gw.types import BatchTensor
+
+
+def taylor_t2_timing_0pn_coeff(total_mass: BatchTensor, eta: BatchTensor):
+    """
+    https://git.ligo.org/lscsoft/lalsuite/-/blob/master/lalsimulation/lib/LALSimInspiralPNCoefficients.c#L1528
+    """
+
+    output = total_mass * G / torch.pow(torch.tensor(C), 3.0)
+    return -5.0 * output / (256.0 * eta)
+
+
+def taylor_t2_timing_2pn_coeff(eta: BatchTensor):
+    """
+    https://git.ligo.org/lscsoft/lalsuite/-/blob/master/lalsimulation/lib/LALSimInspiralPNCoefficients.c#L1545
+    """
+    return 7.43 / 2.52 + 11.0 / 3.0 * eta
+
+
+def taylor_t2_timing_4pn_coeff(eta: BatchTensor):
+    """
+    https://git.ligo.org/lscsoft/lalsuite/-/blob/master/lalsimulation/lib/LALSimInspiralPNCoefficients.c#L1560
+    """
+    return 30.58673 / 5.08032 + 54.29 / 5.04 * eta + 61.7 / 7.2 * eta**2
+
+
+def taylor_t3_frequency_0pn_coeff(total_mass: BatchTensor):
```

Comment:
For completeness, could you put the link in here as well? https://git.ligo.org/lscsoft/lalsuite/-/blob/master/lalsimulation/lib/LALSimInspiralPNCoefficients.c#L1723